google/cloud/bigtable_admin_v2/__init__.py

```diff
@@ -13,262 +13,24 @@
-    "DeleteAppProfileRequest",
-    "DeleteAuthorizedViewRequest",
-    "DeleteBackupRequest",
-    "DeleteClusterRequest",
-    "DeleteInstanceRequest",
-    "DeleteLogicalViewRequest",
-    "DeleteMaterializedViewRequest",
-    "DeleteSchemaBundleRequest",
-    "DeleteSnapshotRequest",
-    "DeleteTableRequest",
-    "DropRowRangeRequest",
-    "EncryptionInfo",
-    "GcRule",
-    "GenerateConsistencyTokenRequest",
-    "GenerateConsistencyTokenResponse",
-    "GetAppProfileRequest",
-    "GetAuthorizedViewRequest",
-    "GetBackupRequest",
-    "GetClusterRequest",
-    "GetInstanceRequest",
-    "GetLogicalViewRequest",
-    "GetMaterializedViewRequest",
-    "GetSchemaBundleRequest",
-    "GetSnapshotRequest",
-    "GetTableRequest",
-    "HotTablet",
-    "Instance",
-    "ListAppProfilesRequest",
-    "ListAppProfilesResponse",
-    "ListAuthorizedViewsRequest",
-    "ListAuthorizedViewsResponse",
-    "ListBackupsRequest",
-    "ListBackupsResponse",
-    "ListClustersRequest",
-    "ListClustersResponse",
-    "ListHotTabletsRequest",
-    "ListHotTabletsResponse",
-    "ListInstancesRequest",
-    "ListInstancesResponse",
-    "ListLogicalViewsRequest",
-    "ListLogicalViewsResponse",
-    "ListMaterializedViewsRequest",
-    "ListMaterializedViewsResponse",
-    "ListSchemaBundlesRequest",
-    "ListSchemaBundlesResponse",
-    "ListSnapshotsRequest",
-    "ListSnapshotsResponse",
-    "ListTablesRequest",
-    "ListTablesResponse",
-    "LogicalView",
-    "MaterializedView",
-    "ModifyColumnFamiliesRequest",
-    "OperationProgress",
-    "OptimizeRestoredTableMetadata",
-    "PartialUpdateClusterMetadata",
-    "PartialUpdateClusterRequest",
-    "PartialUpdateInstanceRequest",
-    "ProtoSchema",
-    "RestoreInfo",
-    "RestoreSourceType",
-    "RestoreTableMetadata",
-    "RestoreTableRequest",
-    "SchemaBundle",
-    "Snapshot",
-    "SnapshotTableMetadata",
-    "SnapshotTableRequest",
-    "StandardReadRemoteWrites",
-    "StorageType",
-    "Table",
-    "Type",
-    "UndeleteTableMetadata",
-    "UndeleteTableRequest",
-    "UpdateAppProfileMetadata",
-    "UpdateAppProfileRequest",
-    "UpdateAuthorizedViewMetadata",
-    "UpdateAuthorizedViewRequest",
-    "UpdateBackupRequest",
-    "UpdateClusterMetadata",
-    "UpdateInstanceMetadata",
-    "UpdateLogicalViewMetadata",
-    "UpdateLogicalViewRequest",
-    "UpdateMaterializedViewMetadata",
-    "UpdateMaterializedViewRequest",
-    "UpdateSchemaBundleMetadata",
-    "UpdateSchemaBundleRequest",
-    "UpdateTableMetadata",
-    "UpdateTableRequest",
-)
-
-from .overlay import *  # noqa: F403
-
-__all__ += overlay.__all__  # noqa: F405
+# iterate and import all submodules to populate sys.modules
+for _, name, _ in pkgutil.walk_packages(
+    path=google.cloud.bigtable.admin.__path__,
+    prefix=_NEW_PATH + ".",
+):
+    mod = importlib.import_module(name)
+    alias = name.replace(_NEW_PATH, __name__, 1)
+    sys.modules[alias] = mod
```

Comment:
I'm curious why this is so different than `google/cloud/bigtable_admin/__init__.py`. Aren't they both aliases?